### ISSUE TITLE
The "Complementary Sampling Rate" described in the paper does not match the code implementation

### ISSUE THREAD

**Issue by z7d1** [hongxin001/open-sampling#5]
The Complementary Sampling Rate describled in your paper:

$$\Gamma_j=(\alpha-\beta_j)/(K\cdot\alpha-1) $$

But in [your implementation](https://github.com/hongxin001/open-sampling/blob/main/alg/ood_noise.py), it does not match:
```python
class OODNoise(OEScratch):
    def __init__(self, args, gpu, num_classes, cls_num_list, train_dataset):
        ...
        if args.alpha > 0:
            self.rebalance_rate = torch.from_numpy(args.alpha - cls_rate).to(self.device)
        ...
    def train_batch_with_out(self, batch_idx, in_set, out_set, epoch):
        ...
        probs = self.rebalance_rate.reshape(1,-1).repeat(out_set[0].shape[0], 1)
        ...
```

Hope for your reply!

**Comment by hongxin001** [comment]
Hi,

The normalisation is calculated in the function of torch.distribution.categorical. So we only need to input the unnormalised probs into the function.

Best wishes
Hongxin

**Comment by z7d1** [comment]
Thanks for your reply!
